app/utils/estimate_prioritization/estimate_prioritization.py

```diff
@@ -121,6 +123,28 @@ def get_prioritized_estimates(estimates: pd.DataFrame,
             selected_estimates.append(get_pooled_estimate(study_estimates))
         else:
             selected_estimates.append(study_estimates)
-    
-    selected_estimate_df = pd.concat(selected_estimates, axis = 1).T.astype(estimates.dtypes.to_dict())
-    return selected_estimate_df
\ No newline at end of file
+
+    if pool:
+        selected_estimate_df = pd.concat(selected_estimates, axis=1).T.astype(estimates.dtypes.to_dict())
+    else:
+        selected_estimate_df = pd.concat(selected_estimates)
+
+    return selected_estimate_df
+
+
+# Gets prioritized estimates without pooling by a specified subgrouping variable
```

Comment:
Could be helpful here to specify that this also returns prioritized estimates that are pooled -- i.e., an overall estimate, as well as the subgroups by that variable. The function name doesn't really make that clear.